packages/react/src/store/idkit.ts

```diff
@@ -13,8 +13,10 @@ import {
 	type ISuccessResult,
 } from '@worldcoin/idkit-core'
 
+const SELF_HOSTED_APP_ID = 'self_hosted'
+
 export type IDKitStore = {
-	app_id: IDKitConfig['app_id']
+	app_id: IDKitConfig['app_id'] | typeof SELF_HOSTED_APP_ID | ''
```

Comment:
why can it be an empty string?

Reply:
why do we require an app_id at all when it's self-hosted? why not just force the app_id to `self-hosted` if the `advanced_selfHosted` param is true?